docs/usage.rst

```diff
@@ -28,16 +38,23 @@ Python library usage
 ====================
 
 The main class is ``silverlabnwb.NwbFile`` defined in ``nwb_file.py``.
-All its methods are documented with docstrings.
+All its methods are documented with docstrings. There are two entry points,
+``import_labview_folder`` and ``create_from_metadata``, which mirror the first
+two command line tools described above.
 
 Quick examples::
 
     from silverlabnwb import NwbFile
 
-    # Write a new file
+    # Write a new file from a LabView folder
     with NwbFile(nwb_path, mode='w') as nwb:
         nwb.import_labview_folder(labview_path)
 
+    # Write a new file from metadata
+    with NwbFile(nwb_path, mode='w') as nwb:
+        nwb.create_from_metadata(metadata_path, user, session_id)
```

Comment:
```suggestion
        nwb.create_from_metadata(metadata_path, user)
```
In fact, we should remove `session_id` from the method altogether, since it's not used!